README.md

```diff
@@ -93,19 +93,26 @@ An intf uses the primitives defined in pkgs to describe the interface between tw
 An intf consists of one or more components, and each component consists of one or more connections.  
 
 components are groups of individual port-to-port connections that constitute a higher-level protocol or logical grouping of signals.
-For example, if two blocks had a val/rdy handshake, the "val" port and the "rdy" port would each be described in a connection, and those connections would be contained within a component.
+Ports describe the shape of an interface (component) and connections describe the repeat of an interface (component).
+For example, if two blocks had a `val/rdy` handshake, the `val` and `rdy` are ports described in `ports` section.
+if this same handshake ports are used multiply to connecting other blocks, those are described inthe `connections` section
```

Comment:
Can you take another pass through your modified section of the readme to ensure that all sentences start with a capital letter (unless the term is surrounded by `ticks` for specific terms) and end with a period? I know it can be annoying some times to figure out where sentence boundaries are in markdown. It might be easier to render the markdown as HTML then use that to review.

Reply:
sure. I wish renderer could automatically capitalize for me...

Reply:
done